domain/src/SharePooling/ValueObjects/QuantityBreakdown.php

```diff
@@ -6,8 +6,9 @@
 
 use Domain\SharePooling\ValueObjects\Exceptions\QuantityBreakdownException;
 use Domain\ValueObjects\Quantity;
+use EventSauce\EventSourcing\Serialization\SerializablePayload;
 
-final class QuantityBreakdown
+final class QuantityBreakdown implements SerializablePayload
```

Comment:
ℹ️ It makes more sense to have value objects implement the same interface for serialisation, instead of `Arrayable`, for instance (which was the case for `FiatAmount` until this PR – see further down)